compiler/rustc_expand/src/mbe/macro_parser.rs

```diff
@@ -40,28 +40,28 @@
 //!
 //! Remaining input: a a a b
 //! cur: [a · $( a )* a b]
-//! Descend/Skip (first item).
+//! Descend/Skip (first mp).
```

Comment:
```rust
//! Descend/Skip (first position).
```
I understand using "mp"s in variable names, but in comments I think we can afford being more wordy.